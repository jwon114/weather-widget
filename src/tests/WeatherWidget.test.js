import React from 'react';
import renderer from 'react-test-renderer';
import WeatherWidget from '../components/WeatherWidget';

test('renders WeatherWidget', () => {
  const tree = renderer
    .create(<WeatherWidget />)
    .toJSON();
  expect(tree).toMatchSnapshot();
});
