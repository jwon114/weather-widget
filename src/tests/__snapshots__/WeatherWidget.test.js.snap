// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders WeatherWidget 1`] = `
<div
  className="weatherApp__container"
>
  <div
    className="weatherApp__loading"
  >
    <img
      alt="loading_spinner"
      src="loading.gif"
    />
  </div>
</div>
`;
